### ISSUE TITLE
Deleted

### ISSUE THREAD

**Issue by Emulator-Team-2** [shadps4-emu/shadPS4#395]

**Comment by raphaelthegreat** [comment]
PR https://github.com/shadps4-emu/shadPS4/pull/370 should fix this, can you confirm?

**Comment by Emulator-Team-2** [comment]
@raphaelthegreat 
The latest build fixes this issue.
https://github.com/shadps4-emu/shadPS4/actions/runs/10332207376

<details> 
<summary>(Screenshots)</summary>

![1](https://github.com/user-attachments/assets/55ffb7c5-4a73-43f5-8ffc-6472d8ece079)
![2](https://github.com/user-attachments/assets/6d212a30-e8cb-49e8-af10-f381431c0e45)

</details>

**Comment by viniciuslrangel** [comment]
On Nvidia GPU it's not able to compile the shaders
```
assert_fail_impl() assert.cpp:13
Shader::IR::Value::U32::<lambda_1>::operator()() value.h:323
Shader::IR::Value::U32() value.h:323
Shader::IR::Value::U32() value.h:321
Shader::Optimization::PatchImageInstruction(Shader::IR::Block &, Shader::IR::Inst &, Shader::Info &, Shader::Optimization::Descriptors &) resource_tracking_pass.cpp:574
Shader::Optimization::ResourceTrackingPass(Shader::IR::Program &) resource_tracking_pass.cpp:609
Shader::TranslateProgram(Common::ObjectPool<…> &, Common::ObjectPool<…> &, span<…>, const Shader::Info &&, const Shader::Profile &) recompiler.cpp:59
Vulkan::PipelineCache::CreateGraphicsPipeline() vk_pipeline_cache.cpp:276
```

**Comment by viniciuslrangel** [comment]
> On Nvidia GPU it's not able to compile the shaders
> 
> ```
> assert_fail_impl() assert.cpp:13
> Shader::IR::Value::U32::<lambda_1>::operator()() value.h:323
> Shader::IR::Value::U32() value.h:323
> Shader::IR::Value::U32() value.h:321
> Shader::Optimization::PatchImageInstruction(Shader::IR::Block &, Shader::IR::Inst &, Shader::Info &, Shader::Optimization::Descriptors &) resource_tracking_pass.cpp:574
> Shader::Optimization::ResourceTrackingPass(Shader::IR::Program &) resource_tracking_pass.cpp:609
> Shader::TranslateProgram(Common::ObjectPool<…> &, Common::ObjectPool<…> &, span<…>, const Shader::Info &&, const Shader::Profile &) recompiler.cpp:59
> Vulkan::PipelineCache::CreateGraphicsPipeline() vk_pipeline_cache.cpp:276
> ```

 #402 fixes the crash

**Comment by Emulator-Team-2** [comment]
Fixed by #414.
